fdbclient/ManagementAPI.actor.cpp

```diff
@@ -999,8 +999,13 @@ ACTOR Future<CoordinatorsResult::Type> changeQuorum( Database cx, Reference<IQuo
 			TEST(old.clusterKeyName() != conn.clusterKeyName());  // Quorum change with new name
 			TEST(old.clusterKeyName() == conn.clusterKeyName()); // Quorum change with unchanged name
 
+			state Reference<ClusterConnectionFile> connFile =
```

Comment:
Change coordinators needs to support hostnames instead of just network addresses

Reply:
After it reads old cluster connection string out of the DB on line 955, the hostnames are resolved there.

Reply:
Now that I am looking at this PR again, adding hostname support for `coodinators` command in fdbcli seems to be orthogonal to the cluster file hostname support. I added a TODO in the code and will create an issue for it.